src/views/widgets/list-widget.php

```diff
@@ -56,9 +56,25 @@
 					 * @param $size
 					 */
 					$thumbnail_size = apply_filters( 'tribe_events_list_widget_thumbnail_size', 'post-thumbnail' );
+
+					/**
+					 * Filters whether the featured image link should be added to the Events List Widget
+					 *
+					 * @since TBD
+					 * @param bool Whether the featured image link should be added or not
+					 */
```

Comment:
should be a line space between `@since` and `@param` like:
```
 * @since TBD
 *
 * @param
```

also, specify the name of the parameter in addition to its type. so something like this:

```
* @param bool $featured_image_link Whether the featured image link should be added or not
```